src/commands/publish/route.rs

```diff
@@ -70,7 +71,8 @@ fn get_routes(user: &GlobalUser, project: &Project) -> Result<Vec<Route>, failur
 
     if !res.status().is_success() {
         let msg = format!(
-            "⛔ There was an error fetching your project's routes.\n Status Code: {}\n Msg: {}",
+            "{} There was an error fetching your project's routes.\n Status Code: {}\n Msg: {}",
+            emoji::WARN,
```

Comment:
I think we're losing the ⛔ emoji here; we should probably add a new emoji to https://github.com/cloudflare/wrangler/blob/master/src/terminal/emoji.rs so we still have it on hand. Perhaps emoji::ERROR?

Reply:
We decided to change that emoji a while back but missed these instances: https://github.com/cloudflare/wrangler/pull/327/files#diff-2e8c637bda89431d960c0efd861afbf9L28

Reply:
Losing the ⛔is intentional to make our warning error messages consistent across the project

Reply:
Gotcha! LGTM then :)